source/extensions/omni.isaac.lab/omni/isaac/lab/envs/direct_rl_env_cfg.py

```diff
@@ -127,3 +127,9 @@ class DirectRLEnvCfg:
 
     Please refer to the :class:`omni.isaac.lab.utils.noise.NoiseModel` class for more details.
     """
+
+    action_bounds: list[float] = [-100, 100]
```

Comment:
Just curious where is [-100, 100] from? I wonder if it's best to leave this user-specified?

Reply:
The 100 limits, comes from our internal codebase, still from legged gym.

I was considering having it None or Inf by default, but then users need to consciously set this value, and I think most people that have training stability issues will probably not think about that. 

Could set it to None and add a FAQ to the docs?

Reply:
I think we can set to -inf, inf